pages/index.js

```diff
@@ -30,10 +32,13 @@ export async function getServerSideProps({ req }) {
       })
     );
 
+    const session = await verifyJwtInRequest(req);
+    const myList = await getMyList({ session });
     const popularList = await getListOfPopularMovies(req);
     const resultsOfPopularList = popularList.results;
     const popularCategory = { id: 1, name: 'Popular', listOfMovies: popularList.results };
-    const listOfCarousels = [popularCategory, ...listOfCategories];
+    const myListCategory = { id: 0, name: 'My List', listOfMovies: myList };
+    const listOfCarousels = [myListCategory, popularCategory, ...listOfCategories];
```

Comment:
Filter out the whole carousel if My List is empty, for example like this:
```suggestion
    const listOfCarousels = myList?.length
      ? [{ id: 0, name: 'My List', listOfMovies: myList }, popularCategory, ...listOfCategories]
      : [popularCategory, ...listOfCategories];
```